lib/restaurant.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'menu'
+require_relative 'dish'
+
+class Restaurant
+
+  attr_reader :menu
+
+  def initialize(menu = nil)
+    @menu = Menu.new(menu)
+  end
+
+  def view_menu
+    @menu.display_menu
```

Comment:
Is display_menu defined anywhere? I can't see it. If you just want to print out the menu contents, as it is in the attribute reader I believe you could just do yourrestaurantinstance.menu